app/helpers/toolbar_helper.rb

```diff
@@ -191,19 +191,20 @@ def toolbar_button_normal(props, view_button = false)
   # attributes in html
   #
   def data_hash_keys(props)
-    hsh = %i(pressed popup console_url prompt explorer onwhen url_parms url).each_with_object({}) do |key, h|
-      h["data-#{key}"] = props[key] if props.key?(key)
+    %i(pressed popup console_url prompt explorer onwhen url_parms url).each_with_object({}) do |key, h|
```

Comment:
`console_url` shouldn't be here anymore I think..? And maybe `window_url` should?